src/enclave/rpc_sessions.h

```diff
@@ -217,7 +243,18 @@ namespace enclave
 
       auto& per_listen_interface = it->second;
 
-      if (
+      if (!certs.contains(listen_interface_id))
```

Comment:
This is a change in behaviour for the "network" endorsement type, which would be the default. Previously, the selfsigned cert would be used and later replaced, but now the interface only accepts sessions when the right endorsement becomes available. I think this makes sense, but worth pointing out.